tmexp/data.py

```diff
@@ -0,0 +1,104 @@
+from collections import Counter
+from typing import Any, Counter as CounterType, DefaultDict, Dict, List, NamedTuple, Set
+
+
+class FileInfo(NamedTuple):
+    blob_hash: str
+    language: str
+
+
+class RefInfo(Dict[str, FileInfo]):
+    pass
+
+
+class FilesInfo(Dict[str, RefInfo]):
+    def __init__(self, refs: List[str]):
+        super().__init__()
+        for ref in refs:
+            self[ref] = RefInfo()
+
+    def remove(self, file_path: str, blob_hash: str) -> None:
+        for ref_dict in self.values():
+            if file_path in ref_dict and blob_hash == ref_dict[file_path].blob_hash:
+                ref_dict.pop(file_path)
+
+
+WordCount = CounterType[str]
```

Comment:
All others are usable as classes (not only type hint). We can do the same here just for consistency :)
```suggestion
class WordCount(CounterType[str]):
    pass
```